src/main/resources/avro/bdg.avdl

```diff
@@ -257,32 +257,43 @@ record AlignmentRecord {
   union { null, Contig } mateContig = null;
 }
 
+enum SequencingPlatform {
+  Illumina
+}
+
+record IlluminaInfo {
+  union { null, string } flowcellId;
+  union { null, int } lane;
+  union { null, int } tile;
+  union { null, int } surface;
+  union { null, int } xPosition;
+  union { null, int } yPosition;
+}
+
 record Sequence {
+  /**
+   Analogous to fastq
+   */
+  union { null, string } sequenceName = null;
   union { null, string } bases = null;
   union { null, string } qualities = null;
+  union { null, SequencingPlatform } platformType = null;
+  union { null, IlluminaInfo } platformInfo = null;
 }
 
 /**
-   The DNA fragment that is was targeted by the sequencer, resulting in
-   one or more reads.
+   A set of objects that should be physically analyzed/stored together.
+   Typically the result of some type of groupby operation.
```

Comment:
One the other hand, @ryan-williams, perhaps there is interest in letting a `Bucket` hold an array of `Buckets`.  Does Avro support this?  May be getting too fancy though, and not clear that there is any immediate demand.